src/components/AvatarGroup/AvatarGroup.jsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import Avatar from "../Avatar/Avatar";
+import Counter from "../Counter/Counter";
+import Tooltip from "../Tooltip/Tooltip";
+import AvatarGroupCounter from "./AvatarGroupCounter";
+import styles from "./AvatarGroup.module.scss";
+
+const AvatarGroup = ({
+  className,
+  avatarClassName,
+  id,
+  children,
+  size,
+  type,
+  max,
+  counterProps,
+  counterTooltipCustomProps
```

Comment:
counterTooltipCustomProps => counterTooltiProps ?

Reply:
I just want people to understand that there are default and custom tooltip content. And when I add counterTooltipIsVirtulizedList back we will need to explain the scope of each prop. If I'll remove "Custom" from the  counterTooltipCustomProps, then I would add "Default" to virtualized prop and that's probably not a good idea :)